### PR TITLE
Multiple Creatives

### PR DESCRIPTION
**Pull request by Pubmatic-Dhruv-Sonone**

### CODE REVIEW

File: vast_4_2.xml [hunk 1]
```diff
@@ -1,46 +1,123 @@
...
+						</MediaFiles>
+						<VideoClicks>
+							<ClickThrough id="blog">
+								<![CDATA[https://iabtechlab.com]]>
+							</ClickThrough>
+						</VideoClicks>
+					</Linear>
+					<UniversalAdId idRegistry="Ad-ID">8465</UniversalAdId>
+				</Creative>
+				<Creative id="5480" sequence="1" adId="2447226">
```
Comment (ShriprasadM) [inline vast_4_2.xml]: Please correct the VAST validation errors and warnings  and benchmark again
Validated using  - https://vastvalidator.iabtechlab.com/dash

Warnings:
Incorrect VAST Version. VAST Version should be - "4.2"

Errors:
Line:Col[103:55]:cvc-complex-type.3.2.2: Attribute 'idValue' is not allowed to appear in element 'UniversalAdId'.
Line:Col[105:18]:cvc-complex-type.4: Attribute 'width' must appear on element 'NonLinear'.
Line:Col[105:18]:cvc-complex-type.4: Attribute 'height' must appear on element 'NonLinear'.

File: README.md [hunk 2]
```diff
@@ -3,16 +3,16 @@
```
Comment (ShriprasadM) [inline README.md:3]: After running `TestXmlEncodingBased` can see following special characters. 
1. Please ensure source VAST XML and 
2. Generated VAST XML are syntactically valid documents
```sh
&#x9;&#x9;<Creative xmlns="http://www.iab.com/VAST" id="5480" sequence="1" adId="2447226">
&#x9;&#x9;  <UniversalAdId xmlns="http://www.iab.com/VAST" idRegistry="Ad-ID" idValue="8465">8465</UniversalAdId>
&#x9;&#x9;  <NonLinearAds xmlns="http://www.iab.com/VAST">
&#x9;&#x9;    <NonLinear xmlns="http://www.iab.com/VAST">
&#x9;&#x9;    <StaticResource xmlns="http://www.iab.com/VAST" creativeType="image/png">
&#x9;&#x9;    http://mms.businesswire.com/media/20150623005446/en/473787/21/iab_tech_lab.jpg
&#x9;&#x9;    </StaticResource>
&#x9;&#x9;    <NonLinearClickThrough xmlns="http://www.iab.com/VAST">
&#x9;&#x9;    http://iabtechlab.com
&#x9;&#x9;    </NonLinearClickThrough>
&#x9;&#x9;    <NonLinearClickTracking xmlns="http://www.iab.com/VAST">
&#x9;&#x9;    http://example.com/trackingurl/clickTracking
&#x9;&#x9;    </NonLinearClickTracking>
&#x9;&#x9;   </NonLinear>
&#x9;&#x9;   <TrackingEvents xmlns="http://www.iab.com/VAST">
&#x9;  &#x9;   <Tracking event="shriprasad"><![CDATA[https://mytracker.com]]></Tracking></TrackingEvents>
&#x9;  &#x9;  </NonLinearAds>
&#x9;&#x9;</Creative>
```

**Review by ShriprasadM — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Pubmatic-Dhruv-Sonone) [reply README.md:3]: Done

File: vast_4_2.xml (hunk 1 above)
Comment (Pubmatic-Dhruv-Sonone) [reply vast_4_2.xml]: Fixed

**Review by ShriprasadM — APPROVED** [review]

**Pull request merged**